cmd/postgres_exporter/postgres_exporter.go

```diff
@@ -18,15 +19,14 @@ import (
 
 	"github.com/blang/semver"
 	"github.com/lib/pq"
+	"github.com/percona/exporter_shared"
```

Comment:
@rnovikovP did you decide about if we need it (for TLS and basic authentication) in this exporter?

Reply:
It already was here in the [master](https://github.com/percona/postgres_exporter/blob/7a7725d490fb19ef0a60d0501d0511635220963b/cmd/postgres_exporter/postgres_exporter.go#L1159-L1160) branch. :)

Reply:
Yes. And Roma raised a question whatever we need it.

Reply:
Sorry for the delay. yes, let's keet it "complicated" until we can simplify it 

added some notes to https://confluence.percona.com/display/PMM/PMM2%3A+Postgres+Exporter